tests/unit/Subscription/OneTimeTest.php

```diff
@@ -4,15 +4,65 @@
 
 namespace GroupLife\Core\tests\Subscription;
 
-use GroupLife\Core\Subscription\Membership;
+use GroupLife\Core\Subscription\OneTime;
+use GroupLife\Core\Visitor;
 use PHPUnit\Framework\TestCase;
 
 class OneTimeTest extends TestCase
 {
+    /**
+     * @throws \Exception
+     */
     public function testScheduleOneTimeTestGetFunctions()
     {
-        $purchase = new Membership(new \DateTimeImmutable('2021-01-01'), new \DateInterval('P1M'));
-        $this->assertEquals(new \DateTimeImmutable('2021-01-01'), $purchase->getStartDay());
-        $this->assertEquals(new \DateInterval('P1M'), $purchase->getPeriod());
+
+        $this->assertEquals(
+            new \DateTimeImmutable('2021-01-01', new \DateTimeZone('Europe/Berlin')),
```

Comment:
Why don't you use system timezone?

Reply:
It is only for tests. TravisCI has UTC timezone

Reply:
Let's make PHPUnit to define the timezone for us, see https://phpunit.readthedocs.io/en/9.5/configuration.html#the-ini-element

Reply:
Fixed